animl-py/src/animl/FileManagement.py

```diff
@@ -27,98 +29,92 @@ def is_image(s):
     ext = os.path.splitext(s)[1].lower()
     return ext in image_extensions
 
-
-#def extractFrames(video,frames,fps):
-#    check metadata 
-    
-#    if(result.frames < 5):
-#        files = data.frame(FilePath = x, Frame = "File Corrupt")
-#    else:
-        
-#      vfilter = "fps=" + round(1 / (result$duration / frames), 3) if len(frames) else "null"
-#      vfilter = "fps=" + str(fps) if len(fps) else vfilter
-        
-#      framerate <- result$video$framerate
-#      tempdir <- tempfile()
-#      dir.create(tempdir)
-    
-#      name <- strsplit(basename(x), ".", fixed = T)[[1]][1]
-        
-#      rnd <- sprintf("%05d", round(stats::runif(1, 1, 99999), 0))
-    
-#      output <- file.path(tempdir, paste0(name, "_", rnd, "_%5d", ".jpg"))
-        
-#      av::av_encode_video(input = x, output = output, framerate = framerate, vfilter = vfilter, verbose = F)
-#      files <- data.frame(FilePath = x, tmpframe = list.files(tempdir, pattern = paste0(name, "_", rnd, "_\\d{5}", ".jpg"), full.names = TRUE), stringsAsFactors = F)
-    
-#      files$Frame <- paste0(outdir, basename(files$tmpframe))
-#      file.copy(files$tmpframe, files$Frame)
-#      file.remove(files$tmpframe)
-#      files[, c("FilePath", "Frame")]
-    
-#    return(files)
-    
-
-
-
-
-def imagesFromVideos(files,outdir = tempfile(),outfile = NA,
-                     format = "jpg",fps = None,frames = None, 
-                     parallel = False, nproc = 1):
-    #if (checkFile(outfile)) { return(loadData(outfile))}
-
-    assert type(files) == "DataFrame", "'files' must be Data Frame."
+def extractImages(file_path, outdir, fps=None, frames=None):
```

Comment:
lets call this imagesFromVideos() since that is what the R version is called

Reply:
imagesFromVideos() is already a function.